HerPublicWebsite.BusinessLogic/Services/RegularJobs/DateHelper.cs

```diff
@@ -16,7 +16,7 @@ public DateHelper(Func<DateTime> getToday)
     public DateTime GetStartOfPreviousMonth()
     {
         var today = getToday();
-        var oneMonthAgo = today.AddMonths(-1);
-        return oneMonthAgo.AddDays(1 - oneMonthAgo.Day); // Remove a month, and set day to 1st
+        var oneMonthAgo = today.AddMonths(-1); // Set day to 1st
```

Comment:
You have put this put this comment on the wrong line. It should be on the line that sets the day to 1st.